src/main/java/no/nav/brregstub/mapper/RolleoversiktMapper.java

```diff
@@ -129,41 +206,50 @@ private static RolleInnehaver mapTilRolleInnhaver(RolleoversiktTo to) {
         return rolleInnehaver;
     }
 
+    private static RolleInnehaver mapTilRolleInnhaver(RsRolleoversikt rsRolleoversikt) {
+        var rolleInnehaver = new RolleInnehaver();
+        rolleInnehaver.setNavn(mapTilNavntype(rsRolleoversikt.getNavn()));
+        Fodselsdato fødselsdato = new Fodselsdato();
+        fødselsdato.setValue(localDateToXmlGregorianCalendar(rsRolleoversikt.getFodselsdato()));
+        rolleInnehaver.setFodselsdato(fødselsdato);
+        rolleInnehaver.setAdresse(mapTilAdresse(rsRolleoversikt.getAdresse()));
+        return rolleInnehaver;
+    }
 
-    private static AdresseType1 mapTilAdresse(AdresseTo to) {
-        if (to == null) {
+    private static AdresseType1 mapTilAdresse(RsAdresse rsAdresse) {
+        if (rsAdresse == null) {
             return null;
         }
         var adresse = new AdresseType1();
-        adresse.setAdresse1(to.getAdresse1());
-        adresse.setAdresse2(to.getAdresse2());
-        adresse.setAdresse3(to.getAdresse3());
-        adresse.setPostnr(to.getPostnr());
-        adresse.setPoststed(to.getPoststed());
+        adresse.setAdresse1(rsAdresse.getAdresse1());
+        adresse.setAdresse2(rsAdresse.getAdresse2());
+        adresse.setAdresse3(rsAdresse.getAdresse3());
+        adresse.setPostnr(rsAdresse.getPostnr());
+        adresse.setPoststed(rsAdresse.getPoststed());
         var land = new AdresseType1.Land();
-        land.setLandkode1(to.getLandKode());
-        land.setValue(to.getLandKode());
+        land.setLandkode1(rsAdresse.getLandKode());
+        land.setValue(rsAdresse.getLandKode());
         adresse.setLand(land);
         return adresse;
     }
```

Comment:
Orika kan gjøre jobben for deg med mapping, men er neppe verdt xtra arbeid å ta inn nå.
Uansett vil bruk av builder der det er mulig som et alternativ redusere atall stmts betraktelig.